pepper-apis/src/main/java/org/broadinstitute/ddp/db/dao/ContentBlockDao.java

```diff
@@ -89,11 +90,22 @@ default void disableContentBlock(long blockId, RevisionMetadata meta) {
             contentDtos = stream.collect(Collectors.toMap(BlockContentDto::getBlockId, Function.identity()));
         }
 
+        Set<Long> templateIds = new HashSet<>();
+        contentDtos.values().forEach(v -> {
+            templateIds.add(v.getBodyTemplateId());
+            if (v.getTitleTemplateId() != null) {
+                templateIds.add(v.getTitleTemplateId());
+            }
+        });
+        Map<Long, Template> templates = getTemplateDao().collectTemplatesByIdsAndTimestamp(templateIds, timestamp);
+
```

Comment:
@yufengwng please check this change: here is added a ContentBlock templates collecting, and then the collected templates are passed to a ContentBlock constructor (see below).

Reply:
Looks about right.